test/indexer/data/aaa96233-bf27-44c7-82df-b4dc15ad4d9d.2018-11-04T113344.698028Z.metadata.json

```diff
@@ -24,6 +24,61 @@
       "update_date": "2018-11-04T10:10:10.908Z"
     }
   },
+  "cell_line_0.json": {
```

Comment:
This change injects a cell line entity that is not referenced in the graph (links.json). Since this canned bundle is used elsewhere in unrelated tests, I'd rather not subvert this otherwise normal bundle by adding that anomaly.

I propose that you make modify this bundle programmatically. Please come talk to me about this so we can brainstorm ways to incorporate the programmatic patching of canned bundles.